Towncrier support in Volto (#3985)
plone/volto: RELEASING.md

Only the current Owners of this organization can grant permissions to the releaser.

### Have a GitHub personal token with permissions to write the Volto's Releases

The `release-it` library takes care of creating and pushing a GitHub Release for each release.
It requires you to get a have GitHub personal token with permissions to write the Volto's Releases.
This can be acquired in your GitHub profile page.
Once you have it, you need to setup in your system a `GITHUB_TOKEN` environment variable set containing it.

```shell
export GITHUB_TOKEN="my_looooong_github_token"
```

See `release-it` documentation of [GitHub releases](https://www.npmjs.com/package/release-it#github-releases) and GitHub documentation [About releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).

### Install `pipx` in your system

The release process calls at some point at `towncrier` which is a Python library using the Python utility `pipx`.
This utility allows you to call and execute Python modules without having to have them installed previously in your system.

> It's like the NodeJS `npx` utility and works pretty much in the same way.

Reviewer: ```suggestion
It works similar to the NodeJS `npx` utility.
```

> If you are in a MacOS system, you can use:

Reviewer: ```suggestion
On macOS, you can install `pipx` into your system:
```

```shell
brew install pipx
```

> or follow more detailed instructions in [`pipx`](https://pypa.github.io/pipx/) documentation page.

Reviewer: ```suggestion
Or follow detailed instructions in the `pipx` documentation for [Installation](https://pypa.github.io/pipx/installation/).
```

## Running the release process

> These are the commands for release Volto:

Reviewer: ```suggestion
These are the commands to make a Volto release:
```

```shell
yarn release
```

> a dry-release command for testing the output is also available:

Reviewer: ```suggestion
A dry-release command for testing the output is also available:
```

```shell
yarn dry-release
```

> and alpha release can also be cut using:

Reviewer: ```suggestion
An alpha release can be cut using:
```
